src/Resources/skeleton/crud/templates/index.tpl.php

```diff
@@ -0,0 +1,27 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1><?= $entity_class_name; ?> index</h1>
+    <table>
```

Comment:
Maybe add `thead` and `tbody` tags? They seem to be pretty popular these days.

Reply:
Haha, I agree - they're super hipster :)